elftools/elf/structs.py

```diff
@@ -347,3 +358,62 @@ def _create_stabs(self):
             self.Elf_half('n_desc'),
             self.Elf_word('n_value'),
         )
+
+    def _create_arm_attributes(self):
+        # Structure of a build attributes subsection header. A subsection is
+        # either public to all tools that process the ELF file or private to
+        # the vendor's tools.
+        self.Elf_Attr_Subsection_Header = Struct('Elf_Attr_Subsection',
+                                                 self.Elf_word('length'),
+                                                 self.Elf_ntbs('vendor_name'),
+        )
+
+        # Structure of a build attribute tag.
+        self.Elf_Attribute_Tag = Struct('Elf_Attribute_Tag',
+                                        Enum(self.Elf_uleb128('tag'),
+                                             **ENUM_ATTR_TAG_ARM)
+        )
+
+
+def _LEB128_reader():
```

Comment:
I'm not sure I understand why this is needed. Can you point to a place where this format is documented?

Reply:
OK, now I see in the spec you provided that LEBs are needed -- but why duplicate all this code with the dwarf files? I suggest placing them in common/construct_utils.py